damus/Models/NotificationsModel.swift

```diff
@@ -144,12 +144,14 @@ class NotificationsModel: ObservableObject, ScrollQueue {
             
             notifs.append(.repost(evid, evgrp))
         }
-        
-        for el in reactions {
-            let evid = el.key
-            let evgrp = el.value
-            
-            notifs.append(.reaction(evid, evgrp))
+
+        if !damus_state.settings.hide_reactions {
```

Comment:
I don't the we need to add this, or change the build_notifications or filter interface. If we're not fetching reactions then the reactions would be empty

Reply:
Fixed.